### ISSUE TITLE
Bug: Hide ugly scrollbars on Windows.

### ISSUE THREAD

**Issue by r002** [mithi/epic-react-exercises#116]
![image](https://user-images.githubusercontent.com/45280066/110229410-2cd29a00-7ed7-11eb-9f56-997980a05c00.png)

![image](https://user-images.githubusercontent.com/45280066/110229483-c00bcf80-7ed7-11eb-8c9b-84fd60585283.png)

When I run your app on my Windows machine in Chrome and Edge, I'm seeing horrible scrollbars that uglyify the GUI.
This should be a simple CSS fix.

I noticed in your [Reddit video](https://www.reddit.com/r/reactjs/comments/lxmeow/react_exercises_with_detailed_solutions/) that the scrollbars don't appear though.  I am guessing you are on Mac?

**Comment by mithi** [comment]
@r002 
I see, that is indeed ugly. Thanks for reporting! Yes I'm on a Mac. I'll have to research how test this though, since I have no access to a windows machine as of the moment, and anytime soon..

**Comment by mithi** [comment]
@r002 This seems like an elegant solution

https://stackoverflow.com/questions/45501705/how-auto-hide-scrollbar-on-windows-browser

```css
.my-elem {
  overflow: hidden
} 

.my-elem:hover {
  overflow: auto
}
```

**Comment by r002** [comment]
> @r002 This seems like an elegant solution
> 
> https://stackoverflow.com/questions/45501705/how-auto-hide-scrollbar-on-windows-browser
> 
> ```css
> .my-elem {
>   overflow: hidden
> } 
> 
> .my-elem:hover {
>   overflow: auto
> }
> ```

@mithi  -- So it's a bit more complicated than this because if we just hid all scrollbars, then they'd disappear entirely, even when we needed them.  (Eg. If the browser window were resized to be very small.)  I ended up restyling the scrollbars entirely in the PR I just submitted.

---

**References:**

- https://stackoverflow.com/questions/4405954/how-do-i-remove-the-horizontal-scrollbar-in-a-div
- https://stackoverflow.com/questions/21563672/google-chrome-vertical-scroll-bars-mysteriously-appear-and-disappear
- https://dev.to/xtrp/how-to-create-a-beautiful-custom-scrollbar-for-your-site-in-plain-css-1mjg
